src/main/python/pybuilder_docker_too/__init__.py

```diff
@@ -182,13 +187,38 @@ def do_docker_run(project, logger, reactor: Reactor):
         time.sleep(3)
 
 
+def _run_docker_compose(project, logger):
+    should_run = project.get_property("run_docker_compose_on_verify", False)
+    compose_file = project.get_property('docker_compose_path_verify', '')
```

Comment:
Perhaps we can put in a smart default here - /src/unittest/docker/test-compose.yml if it exists otherwise get path?  or at least make that the default path ...